wordpress/Makefile

```diff
@@ -59,6 +59,7 @@ app/build:: .build/wordpress/deployer \
                                      $(shell find deployer -type f) \
                                      $(shell find manifest -type f) \
                                      $(shell find versions -type f) \
+																		 | .build/wordpress
```

Comment:
Can you fix formatting here?

Reply:
Thaaat's a tabs vs. spaces bug! Good catch. Fixed.